src/pages/ExamsPage/components/ResetExamAttemptButton.jsx

```diff
@@ -9,8 +8,6 @@ import { useDeleteExamAttempt } from '../hooks';
 
 const ResetExamAttemptButton = ({ username, examName, attemptId }) => {
   const [isOpen, open, close] = useToggle(false);
-  const modalSize = useState('md');
```

Comment:
noticed this causing a bunch of test errors, I don't think we need these to be on state?